browser/brave_wallet/wallet_notification_service.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BRAVE_WALLET_WALLET_NOTIFICATION_SERVICE_H_
+#define BRAVE_BROWSER_BRAVE_WALLET_WALLET_NOTIFICATION_SERVICE_H_
+
+#include <string>
+
+#include "base/memory/raw_ptr.h"
+#include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
+#include "components/keyed_service/core/keyed_service.h"
+#include "mojo/public/cpp/bindings/pending_remote.h"
+#include "mojo/public/cpp/bindings/remote.h"
```

Comment:
These two seems not needed in this header file, and the include for mojo::Receiver is missing.

Reply:
done